packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/configure-self-service-registration-policy/index.md

```diff
@@ -51,6 +51,8 @@ When you enable a self-service registration policy, Okta enforces uniqueness for
         * **User dashboard:** Select to redirect users to your org's Okta homepage.
         * **Custom URL** Select and then enter the URL where you want the authorization server to redirect your users, such as to your custom app or portal.
 
+    > **Note:** Make sure the domain for the custom URL that you've configured for the Default Redirect is added as a [Trusted Origin](/docs/guides/enable-cors/overview/) for redirects.
```

Comment:
```suggestion
    > **Note:** Ensure the domain for the custom URL that you've configured for the Default redirect is added as a [Trusted Origin](/docs/guides/enable-cors/overview/) for redirects.
```

Reply:
I think our style guide prefers "ensure". Also, using "redirect" to reflect the previous parameter name.

Reply:
Fixed. I also bolded "Default redirect" to reflect the parameter name.